### ISSUE TITLE
Restore material property as a superproperty of artMedium, applicable to Product, CreativeWork

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#1294]
Many things beyond VisualArtwork have materials that can be described. We had  http://schema.org/material for VisualArtwork but it was renamed to give us http://schema.org/artMedium.

Proposal here is to bring back 'material' as a superproperty of artMedium but applicable to more types of thing (suggest Product + CreativeWork).  Definition would be "A material that something is made from. "typical values would be: leather, wool, cotton, paper." (Text/URL, to allow for controlled vocabulary). 

(The property would be repeated for multiple values, rather than parsing out comma-separated lists.)

/cc @lazaruscorporation re VisualArtwork and @mfhepp re Product

**Comment by danbri** [comment]
Given that we have already allocated the term /material, I've convinced myself that this is a good idea (and better than wasting it with a redirect). Implemented for review:

http://webschemas.org/material

**Comment by thadguidry** [comment]
@danbri Useful bits from Wikidata

[material used - material the subject is made of or derived from](https://www.wikidata.org/wiki/Property:P186)
[product - material or product produced by a business, industry, facility, or process](https://www.wikidata.org/wiki/Property:P1056)
[source of material - place the material used was mined, quarried, found, or produced](https://www.wikidata.org/wiki/Property:P2647)

More here: https://www.wikidata.org/w/index.php?search=material&title=Special:Search&fulltext=1&searchToken=em9upypo12icyoifkmw74wctz

Some of the See Also properties are:

natural product of taxon
fabrication method
manufacturer
source of material
has part

**Comment by lazaruscorporation** [comment]
I have no problem with this. In fact I think that VisualArtwork should be changed so that it just uses the proposed "material" property inherited from CreativeWork rather than having a specific "artMedium" property.

My reasoning can be seen on the second example I provided on the http://schema.org/VisualArtwork page - Tracey Emin's "My Bed" art installation, where the values of "artMedium" are:

      "artMedium": "bedsheets",
      "artMedium": "condoms",
      "artMedium": "a pair of knickers",
      "artMedium": "pair of slippers",
      "artMedium": "bed"

Simply put, there is no difference (these days) between an "art medium" and a more general "material". 

But if we don't want to deprecate "artMedium" then that's fine - publishers can always choose to use the inherited "material" property" instead as it simply means the same thing.

**Comment by RichardWallis** [comment]
I support the proposal to resurrect material with a domain of CreativeWork and Product - it is not just VisualArtworks that are made from stuff.

My inclination is to keep artMedium as the value it adds is a more art based description of the item.  This could be helped enhanced description e.g:
material: Canvas
artMedium: Watercolour

**Comment by lazaruscorporation** [comment]
Hi Richard

Re. your example, "canvas" would actually be the property of a VisualArtwork's "artworkSurface" property, not it's "artMedium" property.

The typical way a piece of 2-dimensional artwork is described is "medium ON surface" - e.g. Oil on Canvas, Pencil on Paper, Watercolour on Board, etc.  VisualArtwork currently models this using "artMedium" on "artworkSurface", for example:

`<li>Materials:
  <span itemprop="artMedium">oil</span> on <span itemprop="artworkSurface">canvas</span>
</li>`

**Comment by RichardWallis** [comment]
Hi Paul,
Thanks for correcting me on that.

**Comment by lazaruscorporation** [comment]
As such, I think we should extend the Proposal slightly to bring back 'material' as a superproperty of _both_ artMedium _and artworkSurface_ - they're both materials, just playing different roles in a piece of visual artwork.

**Comment by danbri** [comment]
Interesting idea, thanks @lazaruscorporation - let's do that. It will then look almost as if we planned it ;)

Is defining material as  "A material that something is made from, e.g. leather, wool, cotton, paper." adequate for both subproperties?

**Comment by lazaruscorporation** [comment]
Sounds good to me

**Comment by RichardWallis** [comment]
+1

**Comment by danbri** [comment]
Prematurely closed the issue, but I think this is looking good:

http://webschemas.org/material

Please take a look, @chaals @nicolastorzec @shankarnat @tmarshbing @scor @vholland @rvguha (and @mfhepp is on vacation...).

**Comment by danbri** [comment]
I suggest we also edit or add a quick example of a creativework (e.g. sculpture) and/or product, to illustrate the resurrected superproperty. Contributions welcomed...

**Comment by thadguidry** [comment]
@danbri @lazaruscorporation @RichardWallis

Let's also allow artMedium and artworkSurface to expect a type of Product.

That way artists, museums, art historians get manufacturer and other nice properties for describing VisualArtwork's for free.  My wife only does her colored pencil drawings with Prismacolor pencils.  Dr. Pepper bottling company has some art in their museum that only includes Dr. Pepper bottles (not Pepsi or Coca-cola)

**Comment by vholland** [comment]
+1 to the change at http://webschemas.org/material

I also like the idea of extending the range to include Product, so if someone decides to create a sculpture inspired by Andy Warhol's Campbell's Soup Cans, we describe that.

**Comment by thadguidry** [comment]
@vholland Actually you cannot do that currently.  There is no form of 'inspiredBy' property on CreativeWork ( @RichardWallis how did the bib group miss this? ) and that is an added need from what I can see.  Someone open an issue for that please.

**Comment by vholland** [comment]
Right. Currently, I cannot specify the sculpture was inspired by the Warhol piece and I cannot specify I am using Campbell's soup cans as the medium (other than as text).

**Comment by danbri** [comment]
http://schema.org/isBasedOn is in that direction but with a more concrete and less inspirational tone.

**Comment by RichardWallis** [comment]
The Bib group I seem to remember were not inspired to come up with such a term, which is not a common one in bibliographic metadata I believe.  isBasedOn would be more appropriate there.

I personally agree that inspiredBy would be a useful addition. 

In isolation, does extending the range of material to include Product make sense or are we being a bit edge-case here? Or are we looking for another subtype of material createdWith? createdFrom? (with  Product in its range)?

**Comment by danbri** [comment]
It does feel a little corner-case, but if it also helps bridge these two fairly separate corners of schema.org it could be of value. Are there any art-making-and-sharing sites where people catalogue their materials in such obsessive detail that this product data could actually be available?

**Comment by lazaruscorporation** [comment]
I'd rather not add another createdWith/createdFrom subtype of material - I would have thought that 'material' (or the more specialised 'artMedium' for VisualArtwork) is enough - the value of 'artMedium' can be set as a Text value of "Dr. Pepper bottles"

> Are there any art-making-and-sharing sites where people catalogue their materials in such obsessive detail that this product data could actually be available?

Not obsessively by brand, but they do by controlled vocabulary (which is already permitted, as described in my blog post on integrating schema.org/VisualArtwork with the Getty AAT Vocabs (esp. artMaterial and artworkSurface) - http://www.lazaruscorporation.co.uk/blogs/arts-tech/posts/getty-aat-linked-open-data-in-schemadotorg-visualartwork)

Here's a live example (of mine!) that actually does name the brand of paper: http://www.lazaruscorporation.co.uk/artists/paul-watson/drawings/the-procession-drawing

and here's the relevant markup:

`<span property="schema:artMedium" resource="http://vocab.getty.edu/aat/300022414">Charcoal</span> on <span property="schema:artworkSurface" resource="http://vocab.getty.edu/aat/300014172">Paper (Fabriano 'Five' 300gsm)</span>`

The controlled vocabulary link is just to the record for "printing paper", but the text value of the artworkSurface does mention the paper brand (and its weight).

**Comment by thadguidry** [comment]
@danbri Yes, there are many.  Mostly museums, like The Met, have all that data and include it online sometimes (with our without markup).  Here's an example where you have an artMedium that is a cigarette pack...who's manufactured by Hassan Cigarettes way back in the day.

http://www.metmuseum.org/art/collection/search/397122?sortBy=Relevance&amp;what=Baseball+cards&amp;ft=*&amp;pg=1&amp;rpp=20&amp;pos=21

Watches are another. Specifically Pocket Watches that were etched or inscribed by an artist, where some museums have the artMedium as the watch manufacturer and the artist name who designed or inscribed the scene on the back of the watch.

And the other way around, but not directly using artMedium but instead MTE...Herman Miller furniture is an MTE with SFMOMA in their database (but not shown in web schema), in that they class some of the furniture as both Art and Furniture.

**Comment by lazaruscorporation** [comment]
@thadguidry I guess once you're out of the realm of fine art and into collectables then maybe brand is more important.

In the 1st Met Museum link the artMedium is not a cigarette pack (according to the Met site) but rather "Commercial lithographs with half-tone photograph". The artworkSurface (not listed on the page) would probably be "cardstock". It's the Publisher (already a property of CreativeWork and VisualArtwork) that is Hassan Cigarettes.

However I can't dispute the fact that 2 of the materials in the other 2 links - Bakelite and Lurex - are brand names of materials.

So if you're convinced that we should extend the range of possible values of "artMedium" and "artSurface" to include Product in addition to Text and URL then I certainly won't object.

However I'd suggest that we define the expected types in the proposed superproperty "material" (material would therefore expect Text, URL, or Product), and these expected types will then automatically be inherited by its child properties of "artMedium" and "artSurface".

**Comment by danbri** [comment]
Yup makes sense to do it at the 'material' level.

**Comment by danbri** [comment]
Ok, Product added as a value for "material": http://webschemas.org/material

I think we're done, although an example would be welcome.

**Comment by thadguidry** [comment]
@danbri And to that end, we really don't explain usage of sub properties or super properties.... so #1301

**Comment by nicolastorzec** [comment]
No issue with restoring "material" and its sub properties.

In addition, I'm with @vholland regarding "isInspiredBy": i.e. beyond [isBasedOn](https://schema.org/isBasedOn), adding something like "isInspiredBy" would be interesting and useful to explicitly link related entities together. 

Initial use case:
- Goal: Capture that artists and creative works can be inspired by other artists and works
- Domain: Person, PerformingGroup, CreativeWork
- Range: Person, PerformingGroup, CreativeWork

Expanded use case:
- Domain/Range could be expanded to any artifact and most things thing
- E.g. products inspired by another products, buildings inspired by sculptures or land formations...
